File: examples/c/constructs/for_loop.c
// Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
int main(void) {
    long i = 0;
    while (i < 10) {
        printf("%ld", i);
        if (i < 9) {
            printf(", ");
        }
        i = i + 1;
    }
    puts("");
    return 0;
}

File: examples/ruby/constructs/for_loop.rb
# Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
for i in 0...10 do
    print(i)
    if i < 9 then
        print(", ")
    end
end
puts("")
